README.md

```diff
@@ -3,53 +3,57 @@
-* Generate skeleton chaincode projects
+The IBM Blockchain extension allows a user to easily write and test smart contract projects in Visual Studio Code:
+* Generate skeleton smart contract projects
 * Adds a Fabric view to connect and interact with Fabric Blockchain networks
 
+## Features
+The IBM Blockchain extension provides an explorer and commands accessible from the Command Palette, for developing smart contracts quickly:
+<!---Table of commands with columns: 'command' and 'description'
+--->
 
-## Connecting to your own Hyperledger Fabric instance
-
-Using this extension, you can connect to a pre-configured local instance of Hyperledger Fabric, or you can connect to your own Hyperledger Fabric instance. If you choose to connect to your own Fabric instance, it must be Fabric 1.3.0 or later.
-
-**When using a local Hyperledger Fabric instance the extension will automatically pull and tag the required Docker images.**
-
-To connect to your own Hyperledger Fabric instance within the extension, you must first pull and tag the Fabric 1.3.0 Docker images by running the following commands:
-
-```
-docker pull nexus3.hyperledger.org:10001/hyperledger/fabric-ca:amd64-1.3.0-stable
-docker tag nexus3.hyperledger.org:10001/hyperledger/fabric-ca:amd64-1.3.0-stable hyperledger/fabric-ca
-
-docker pull nexus3.hyperledger.org:10001/hyperledger/fabric-orderer:amd64-1.3.0-stable
-docker tag nexus3.hyperledger.org:10001/hyperledger/fabric-orderer:amd64-1.3.0-stable hyperledger/fabric-orderer
-
-docker pull nexus3.hyperledger.org:10001/hyperledger/fabric-peer:amd64-1.3.0-stable
-docker tag nexus3.hyperledger.org:10001/hyperledger/fabric-peer:amd64-1.3.0-stable hyperledger/fabric-peer
-
-docker pull nexus3.hyperledger.org:10001/hyperledger/fabric-tools:amd64-1.3.0-stable
-docker tag nexus3.hyperledger.org:10001/hyperledger/fabric-tools:amd64-1.3.0-stable hyperledger/fabric-tools
-
-docker pull nexus3.hyperledger.org:10001/hyperledger/fabric-ccenv:amd64-1.3.0-stable
-docker tag nexus3.hyperledger.org:10001/hyperledger/fabric-ccenv:amd64-1.3.0-stable hyperledger/fabric-ccenv
-
-```
+| Command | Description |
+| --- | --- |
+| Add Connection | Add a Hyperledger Fabric instance connection |
+| Add Identity To Connection | Add an identity to be used when connecting to a Hyperledger Fabric instance  |
+| Connect To Blockchain | Connect to a Hyperledger Fabric blockchain using a blockchain connection |
+| Create Smart Contract Project | Create a new JavaScript or TypeScript smart contract project |
+| Delete Connection | Delete a Hyperledger Fabric instance connection |
+| Delete Package | Delete a smart contract package |
+| Disconnect From Blockchain | Disconnect from the blockchain you're currently connected to |
+| Install Smart Contract | Install a smart contract package onto a peer |
+| Instantiate Smart Contract | Instantiate an installed smart contract package <br>**Note: This currently doesn't work with IBP instances*|
```

Comment:
"Instantiate an installed smart contract package **onto a channel**" (since we specify the install target on the install command).

"IBM Blockchain Platform" instead of "IBP".  Damn, it sounds bad to say "the IBM Blockchain Platform extension doesn't work with IBM Blockchain Platform", doesn't it haha.  But, yeah, it's true, so...